### DISCUSSION TITLE
post/sre-re-pylon/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#48] [Announcements]
# post/sre-re-pylon/

카카오페이의 레거시 배포 환경에서 개선한 내용을 소개합니다.

https://tech.kakaopay.com/post/sre-re-pylon/

**Comment by smilejj91** [comment]
좋은 글 감사합니다 😃

**Comment by Hoo-sung** [comment]
좋은 글 잘 보았습니다.!! RE파트에서 하는 일에 대해 구체적으로 알 수 있었습니다.😀
